packages/teams-js/src/public/logger.ts

```diff
@@ -0,0 +1,21 @@
+import { GlobalVars } from '../internal/globalVars';
+
+export namespace logger {
+  /**
+   * todo: doc
+   * @returns bool
+   */
+  export function turnOnConsoleLog(): boolean {
```

Comment:
probably no reason you need to return these booleans

Reply:
Updated. Change `boolean` to `void` now.